dist/tools/shellcheck/check.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2018 Freie Universität Berlin
+#                    Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+SHELLCHECK_CMD="$(which shellcheck)"
```

Comment:
I thought the agreed upon and portable way to check if a command is available was determined to be `command -v`.

Reply:
good point, oversaw that one

Reply:
So remove the `which shellcheck` and assume that executing `shellcheck` works?

Reply:
No, just use `command -v` instead of which ;-)

Reply:
Something like that is already [here](https://github.com/RIOT-OS/RIOT/pull/9063/files#diff-ff4f19db62b9b344ac5cfcd01b5e4716R32) :-)

Reply:
Never mind, I was not aware that `command` is an actual command and that it can be used as an alternative for `which` :expressionless: